packer/heroku/squidex/squidex.sh

```diff
@@ -10,4 +10,6 @@ export STORE__MONGODB__CONTENTDATABASE="Squidex"
 
 export ASPNETCORE_URLS="http://+:$PORT"
 
+export IDENTITY__GOOGLECLIENT=""
+
```

Comment:
I think we have to do it as well for all other cloud providers like GITHUB and MICROSOFT